process_preds.Rmd

```diff
@@ -74,11 +79,15 @@ process_preds <- function(predictor_path, template, resolution, save, output_pat
     # Project cropped raster into template CRS
     raster_file <- project(raster_file, crs(template), res = resolution)
     
+    if(distance == TRUE) {
+      raster_file <- terra::distance(raster_file)
```

Comment:
I'm realizing we might need to test this for input files that are already rasters. The `distance()` function needs the raster to have NA values for any cell that is not the target, which is the case for the outputs of `rasterize()` above but not sure all existing .tifs will be formatted that way. They might have 0 for non-target values instead of NA.